lib/MusicBrainz/Server/Form/Admin/Attributes/Language.pm

```diff
@@ -37,9 +40,18 @@ has_field 'iso_code_3' => (
 );
 
 has_field 'frequency' => (
-    type => '+MusicBrainz::Server::Form::Field::Integer',
+    type => 'Select',
+    required => 1
 );
 
+sub options_frequency {
+    return [
+        $LANGUAGE_FREQUENCY_HIDDEN, lp('Hidden', 'language_script_frequency'),
+        $LANGUAGE_FREQUENCY_OTHER, lp('Other', 'language_script_frequency'),
+        $LANGUAGE_FREQUENCY_FREQUENT, lp('Frequent', 'language_script_frequency'),
```

Comment:
1. There is no need to put underscores in message context, it supports spaces;
2. Terms `HIDDEN` and `OTHER` refer to “display group” but are used for “frequency”, it might be more appropriate to replace these with `UNUSED` and `RARE` respectively;
3. Label 'Frequent' could probably be replaced with 'Frequently used' as in [Form::Utils](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-06-03/lib/MusicBrainz/Server/Form/Utils.pm#L60).

Reply:
I replaced frequent but I'd leave other, specifically so we can also reuse the other 'language optgroup' string (instead of adding a new context for this).